contracts/Market.sol

```diff
@@ -80,4 +101,16 @@ contract Market is UseState {
 		issuedMetrics += 1;
 		return address(metrics);
 	}
+
+	function getTotalVotes() public view returns (uint256) {
```

Comment:
Do you change to `public` field this getter method? ( Following the past PR #12  )

Reply:
You're right.
I fix this.

Reply:
I fixed this.